Formula/gnuradio.rb

```diff
@@ -31,17 +42,8 @@ class Gnuradio < Formula
   depends_on "sdl" => :recommended
   depends_on "jack" => :recommended
   depends_on "portaudio" => :recommended
-
-  # gnuradio is known not to compile against CMake >3.3.2 currently.
-  resource "cmake" do
-    url "https://cmake.org/files/v3.3/cmake-3.3.2.tar.gz"
-    sha256 "e75a178d6ebf182b048ebfe6e0657c49f0dc109779170bad7ffcb17463f2fc22"
-  end
-
-  resource "numpy" do
-    url "https://pypi.python.org/packages/source/n/numpy/numpy-1.10.1.tar.gz"
-    sha256 "8b9f453f29ce96a14e625100d3dcf8926301d36c5f622623bf8820e748510858"
-  end
+  depends_on "pygtk" => :recommended
+  depends_on "wxpython" => :recommended
```

Comment:
I disabled both GUI libraries by default.

But it doesn't make sense to pick only one. Gnuradio-companion, the main gnuradio GUI, visual signal flow editor, uses GTK, however GUI blocks (for example that display spectrogram or allow to change parameters as GUI slider) use either WXWidgets or Qt (seems that most blocks use wx). I think most users who need gnuradio-companion need all three toolkits or at least gtk + wx.